packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -340,12 +360,14 @@ export default Component.extend({
      * @action updateColumnDisplayName
      */
     updateColumnDisplayName(column, displayName) {
-      this.attrs.onUpdateReport(
-        'updateColumn',
-        assign({}, column, {
-          displayName
-        })
-      );
+      run.scheduleOnce('afterRender', () => {
```

Comment:
What changed here? Double update in a render loop?

Reply:
For some reason with the vertical collection tests were failing with `You modified "columns" twice in a single render.`
Scheduling it in a separate render loop seemed to solve the issue. Did you have another suggestion?
I'll admit I have to read more about the run loop and understand how it works